draft-ietf-quic-recovery.md

```diff
@@ -326,10 +326,10 @@ the peer. For Initial and Handshake packets, a max_ack_delay of 0 is used.
 At a high level, an endpoint measures the time from when a packet was sent to
 when it is acknowledged as a round-trip time (RTT) sample.  The endpoint uses
 RTT samples and peer-reported host delays ({{host-delay}}) to generate a
-statistical description of the connection's RTT.  An endpoint computes the
+statistical description of the path's RTT.  An endpoint computes the
 following three values: the minimum value observed over the lifetime of the
-connection (min_rtt), an exponentially-weighted moving average (smoothed_rtt),
-and the variance in the observed RTT samples (rttvar).
+congestion control context (min_rtt), an exponentially-weighted moving average
```

Comment:
```suggestion
RTT estimator's context (min_rtt), an exponentially-weighted moving average
```

Reply:
We never define this, but isn't the RTT estimator a subset of the congestion control context?  Also, we only use RTT estimator once in this document at the moment.

Possibly I should use path for all of these now, against @erickinnear suggestion?  Also, path is much shorter than "congestion control context"